plugins/postcss-global-data/test/basic.css

```diff
@@ -0,0 +1,35 @@
+@media (--mq-a) {
+	body {
+		order: 1;
+	}
+}
+
+@media (--mq-b) {
+	body {
+		order: 2;
+	}
+}
```

Comment:
Maybe sufficient to use simpler `@custom-media` examples?

Nothing incorrect about these, but a simpler example is less likely to require changes if the `postcss-custom-media` produces slightly different output.

Reply:
This is a great suggestion, I've simplified these even further